### ISSUE TITLE
Font problem with ggpairs correlations not showing

### ISSUE THREAD

**Issue by robjhyndman** [ggobi/ggally#378]
``` r
library(GGally)
#> Loading required package: ggplot2
#> Registered S3 method overwritten by 'GGally':
#>   method from   
#>   +.gg   ggplot2
ggpairs(flea, columns = 2:4)
```

![](https://i.imgur.com/CMtLFh6.png)

<sup>Created on 2020-08-15 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

<details>

<summary>

Session info

</summary>

``` r
sessionInfo()
#> R version 4.0.2 (2020-06-22)
#> Platform: x86_64-pc-linux-gnu (64-bit)
#> Running under: Ubuntu 20.04.1 LTS
#> 
#> Matrix products: default
#> BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0
#> LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0
#> 
#> locale:
#>  [1] LC_CTYPE=en_AU.UTF-8       LC_NUMERIC=C              
#>  [3] LC_TIME=en_AU.UTF-8        LC_COLLATE=en_AU.UTF-8    
#>  [5] LC_MONETARY=en_AU.UTF-8    LC_MESSAGES=en_AU.UTF-8   
#>  [7] LC_PAPER=en_AU.UTF-8       LC_NAME=C                 
#>  [9] LC_ADDRESS=C               LC_TELEPHONE=C            
#> [11] LC_MEASUREMENT=en_AU.UTF-8 LC_IDENTIFICATION=C       
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices datasets  utils     methods   base     
#> 
#> other attached packages:
#> [1] GGally_2.0.0  ggplot2_3.3.2
#> 
#> loaded via a namespace (and not attached):
#>  [1] Rcpp_1.0.5         pillar_1.4.6       compiler_4.0.2     RColorBrewer_1.1-2
#>  [5] highr_0.8          plyr_1.8.6         tools_4.0.2        digest_0.6.25     
#>  [9] evaluate_0.14      lifecycle_0.2.0    tibble_3.0.3       gtable_0.3.0      
#> [13] pkgconfig_2.0.3    rlang_0.4.7        curl_4.3           yaml_2.2.1        
#> [17] xfun_0.16          xml2_1.3.2         httr_1.4.2         withr_2.2.0       
#> [21] stringr_1.4.0      dplyr_1.0.1        knitr_1.29         generics_0.0.2    
#> [25] vctrs_0.3.2        grid_4.0.2         tidyselect_1.1.0   reshape_0.8.8     
#> [29] glue_1.4.1         R6_2.4.1           rmarkdown_2.3      farver_2.0.3      
#> [33] purrr_0.3.4        magrittr_1.5       scales_1.1.1       ellipsis_0.3.1    
#> [37] htmltools_0.5.0    mime_0.9           colorspace_1.4-1   renv_0.11.0       
#> [41] labeling_0.3       stringi_1.4.6      munsell_0.5.0      crayon_1.3.4
```

</details>

Perhaps the package assumes some fonts are installed and available. There doesn't seem to be a way to select the font in the function call.

**Comment by dicook** [comment]
See issue 377 (the next one). There is a work around for the moment. 

It seems to be a linux issue, and doesn't reproduce on OSX.

**Comment by robjhyndman** [comment]
Closed as duplicate of #377
